site/e2e/tests/auditLogs.spec.ts

```diff
@@ -13,62 +13,63 @@ test.describe.configure({ mode: "parallel" });
 
 test.beforeEach(async ({ page }) => {
 	beforeCoderTest(page);
-	await login(page, users.auditor);
 });
 
-async function resetSearch(page: Page) {
+async function resetSearch(page: Page, username: string) {
```

Comment:
Nit: I think a better name for this function could be `filterByUser` or `searchByUser`.